pymobiledevice3/lockdown.py

```diff
@@ -118,6 +120,100 @@ def __init__(self, udid=None, client_name=DEFAULT_CLIENT_NAME):
         self.paired = True
         return
 
+    def forward(self, src_port: int, dst_port: int):
+        """
+        forward each connection from given local machine port to remote device port
+        """
+        # dictionaries containing the required maps to transfer data between each local
+        # socket to its remote socket and vice versa
+        remote_to_local = {}
```

Comment:
Perhaps use the same dictionary for both?

Reply:
ok